bundle/manifests/authorino-operator-authorino-authconfig-editor-role_rbac.authorization.k8s.io_v1_clusterrole.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   creationTimestamp: null
-  name: authorino-authconfig-editor-role
+  name: authorino-operator-authorino-authconfig-editor-role
```

Comment:
The bundle includes the authorino cluster roles, but now that we are generating them via kustomize it's getting the namespace prefix we are setting in the default config. I don't think this is an issue, but if it we can still do it this way, but would need to use a different config other than default.

Reply:
This will require changes in the reconciler as it needs to know the name of these roles to create the bindings for the instance SAs: https://github.com/Kuadrant/authorino-operator/blob/26a1140e3e927cf567dbe1c54e98344e4ad681c6/controllers/authorino_controller.go#L51

Reply:
OK, will look at doing this differently so it doesn't alter the names of the authorino resources.